### PR TITLE
DHFPROD-6194:Recognize errors in Target Permissions on field blur

### PR DESCRIPTION
**Pull request by srinathgit**
### Description
Recognize errors in Target Permissions on field blur
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [x] Added Tests
  

- ##### Reviewer:

- [x] Reviewed Tests

### CODE REVIEW

File: marklogic-data-hub-central/ui/src/components/advanced-settings/advanced-settings-dialog.test.tsx [hunk 1]
```diff
@@ -235,8 +235,16 @@ describe('Advanced Step Settings dialog', () => {
     //https://github.com/testing-library/react-testing-library/issues/375
     //Solution in github wont work because our list for additional collection is empty to start with
 
-    fireEvent.change(getByPlaceholderText('Please enter target permissions'), { target: { value: 'permissions-changed' }});
-    expect(getByPlaceholderText('Please enter target permissions')).toHaveValue('permissions-changed');
+    fireEvent.change(getByPlaceholderText('Please enter target permissions'), { target: { value: 'data-hub-operator' }});
+    expect(getByPlaceholderText('Please enter target permissions')).toHaveValue('data-hub-operator');
+    fireEvent.blur(getByPlaceholderText('Please enter target permissions'));
+    expect(getByTestId('validationError')).toHaveTextContent('The format of the string is incorrect. The required format is role,capability,role,capability,....');
```
Comment (wooldridge) [inline marklogic-data-hub-central/ui/src/components/advanced-settings/advanced-settings-dialog.test.tsx:241]: Let's fix the ellipsis here, it should be three dots (...) not four. (Four dots suggests the string should end in a period, which is incorrect.) And fix it in the place that is being tested: config/messages.config.ts, line 3

**Review by wooldridge — CHANGES_REQUESTED** [review]

**Review by briantang — CHANGES_REQUESTED** [review]
@srinathgit functionality looks good but I'm seeing related RTL tests failing in other files: 

-mapping-card.test.tsx: Mapping Card component › Open Advanced Step settings
-load-list.test.tx: Load data component › Verify Load settings from list view renders correctly

Could you refactor those respective test files as well?

**Comment by srinathgit** [comment]
> @srinathgit functionality looks good but I'm seeing related RTL tests failing in other files:
> 
> -mapping-card.test.tsx: Mapping Card component › Open Advanced Step settings
> -load-list.test.tx: Load data component › Verify Load settings from list view renders correctly
> 
> Could you refactor those respective test files as well?

Fixed the failing tests in the latest commit.

**Review by briantang — APPROVED** [review]
Looks good now.

**Review by wooldridge — APPROVED** [review]
Looks good: https://drive.google.com/file/d/1mfXvySK8cC8toYAGD5oTQ-wm-QsxEx-i/view

**Pull request merged**